hw/xbox/nv2a/nv2a_psh.c

```diff
@@ -177,21 +185,21 @@ struct InputVarInfo {
 struct FCInputInfo {
     struct InputInfo a, b, c, d, e, f, g;
     int c0, c1;
-    //uint32_t c0_value, c1_value;
+    // uint32_t c0_value, c1_value;
     bool c0_used, c1_used;
     bool v1r0_sum, clamp_sum, inv_v1, inv_r0, enabled;
 };
 
 struct OutputInfo {
-    int ab, cd, muxsum, flags, ab_op, cd_op, muxsum_op,
-        mapping, ab_alphablue, cd_alphablue;
+    int ab, cd, muxsum, flags, ab_op, cd_op, muxsum_op, mapping, ab_alphablue,
+        cd_alphablue;
```

Comment:
Why did we ever decide to put these all in one line anyway?
In my opinion we should probably split these into multiple lines anyway (maybe grouped slightly like in `FCInputInfo`)